android/app/src/main/java/org/prideinlondon/festival/MainApplication.java

```diff
@@ -33,8 +33,8 @@ public boolean getUseDeveloperSupport() {
     protected List<ReactPackage> getPackages() {
       return Arrays.<ReactPackage>asList(
           new MainReactPackage(),
-            BugsnagReactNative.getPackage(),
-            new RNReactNativeHapticFeedbackPackage(),
+            BugsnagReactNative.getPackage(),	
```

Comment:
Can we just quickly get rid of these spaces so that this file doesn't show up as changed

Reply:
yes I'll do that right away